crates/net/discv5/src/network_stack_id.rs

```diff
@@ -21,7 +21,7 @@ impl NetworkStackId {
     pub const OPSTACK: &'static [u8] = b"opstack";
 
     /// Returns the [`NetworkStackId`] that matches the given [`ChainSpec`].
-    pub const fn id(chain: &ChainSpec) -> Option<&'static [u8]> {
```

Comment:
pls revert

Reply:
This currently can only be `const` when the optimism feature is disabled. Do we want this to be defined conditionally?

Reply:
ok, thanks for the explanation